abtesting/Design/SYMBOLS.md

```diff
@@ -0,0 +1,172 @@
+# A/B Testing Quickstart SwiftUI Symbols
+
+## Table of Contents
+- [ABTestingExampleApp](#abtestingexampleapp)
+    - [appConfig](#appconfig)
+    - [init](#init)
+    - [body](#body)
+- [AppConfig](#appconfig-1)
+    - [AppConfig](#appconfig-2)
+        - [colorScheme](#colorscheme)
+        - [init](#init-1)
+        - [deinit](#deinit)
+        - [updateFromRemoteConfig](#updatefromremoteconfig)
+        - [updateFromRemoteConfigAsync](#updatefromremoteconfigasync)
+        - [printInstallationAuthToken](#printinstallationauthtoken)
+    - [ColorScheme](#colorscheme-1)
+        - [init](#init-2)
+    - [RemoteConfigFetchAndActivateStatus](#remoteconfigfetchandactivatestatus)
+        - [debugDescription](#debugdescription)
+- [ContentView](#contentview)
+    - [ContentView](#contentview-1)
+        - [appConfig](#appconfig-3)
+        - [data](#data)
+        - [body](#body-1)
+    - [BasicList](#basiclist)
+        - [data](#data-1)
+        - [body](#body-2)
+
+## ABTestingExampleApp
+```swift
+@main
+struct ABTestingExampleApp: App
+```
+main point of entry into app
+
+### appConfig
```

Comment:
I left this idea on other PR as well:
Optional: consider making header an inline code block
>### `appConfig`
>```swift
>var appConfig: AppConfig
>```
>stores the AppConfig instance

Reply:
Done ✅